guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1109 @@
+# Updating from v12 to v13
+
+## Before you start
+
+v13 requires Node 14.x or higher to use, so make sure you're up to date. To check your Node version, use `node -v` in your terminal or command prompt, and if it's not high enough, update it! There are many resources online to help you with this step based on your host system.
+
+Once you've got Node up-to-date, you can install v13 by running `npm install discord.js` in your terminal or command prompt for text-only use, or `npm install discord.js @discordjs/opus` for voice support.
+
+You can check your discord.js version with `npm list discord.js`. Should it still show v12.x, uninstall (`npm uninstall discord.js`) and re-install discord.js and make sure the entry in your package.json does not prevent a major version update. Please refer to the [npm documentation](https://docs.npmjs.com/files/package.json#dependencies) for this.
+
+## API Version
+
+Discord.js v13 makes the switch to Discord API v9! In addition to this, the new major version also includes a bunch of cool new features...
+
+## Slash Commands
+
+Discord.js now has support for Slash Commands!
+
+Refer to the [Slash Commands](/interactions/registering-slash-commands) section of this guide to get started.
+
+In addition to the `interactionCreate` event covered in the above guide, this release also includes the new Client events `applicationCommandCreate`, `applicationCommandDelete` and `applicationCommandUpdate`.
+
+## Message Components
+
+Discord.js now has support for Message Components!
+
+This introduces the `MessageActionRow`, `MessageButton` and `MessageSelectMenu` classes, as well as associated interactions and collectors. 
+
+Refer to the [Message Components](/interactions/buttons) section of this guide to start using Message Components.
+
+## Threads
+
+Discord.js now has support for Threads! Threads are a new type of sub-channel that can be used to help separate conversations into a more meaningful flow.
+
+This introduces the `ThreadManager` class, which can be found as `TextChannel#threads`, in addition to `ThreadChannel`, `ThreadMemberManager` and `ThreadMember`.
+
+There are also five new events; `threadCreate`, `threadDelete`, `threadListSync`, `threadMemberUpdate` and `threadMembersUpdate`.
+
+Refer to the [Threads](/popular-topics/threads) section of this guide to start using Threads.
+
+## Voice
+
+Support for voice has been separated into its own module. You now need to install and use [@discordjs/voice](https://github.com/discordjs/voice) for interacting with the Discord Voice API.
+
+## Customizable Manager Caches
+
+A popular request that has finally been heard - the `Client` class supports a new option, `makeCache` which accepts a `CacheFactory`.
+
+By combining this with the helper function `Options.cacheWithLimits` users can define custom caps on the caches of each Manager, and let discord.js handle the rest.
```

Comment:
Add a comma punctuation, and use the non-abbreviated word.
```suggestion
By combining this with the helper function `Options.cacheWithLimits`, users can define custom capicities on the caches of each Manager, and let discord.js handle the rest.
```

Reply:
* capacities

Reply:
A cap is a limit, not a capacity 🤔

Reply:
Alright, fixed the typo. But should it be limit or capacity? I'm pretty sure the one ImRodry gave is just the same as above,
![921DDD34-ECF9-4196-95CE-7F25A75990CB](https://user-images.githubusercontent.com/77086272/125071517-f5296800-e0eb-11eb-856e-ead245bcd39c.jpeg) via https://www.merriam-webster.com/dictionary/cap.

Reply:
Capacity just wouldn't make much sense in this scenario. The way I see it is that this option allows you to limit what you want to cache, therefore caps is correct

Reply:
"users can define limits on each Manager's cache" would clear up this confusion

Reply:
I think it’s fine as it is

Reply:
> "users can define limits on each Manager's cache" would clear up this confusion

Got it, I changed it, thanks.